docs/usage.rst

```diff
@@ -239,6 +239,36 @@ And set an ``Access-Control-Allow-Origin`` HTTP header on that file.
 .. note:: both of these steps need to be done or your scripts might not
    even get executed
 
+Promises
+--------
+
+By default, Raven.js does not capture unhandled promise rejections.
+
+Most Promise libraries have a global hook for capturing unhandled errors. You will need to
+manually hook into such an event handler and call ``Raven.captureException`` or ``Raven.captureMessage``
+directly.
+
+For example, the `RSVP.js library
+<https://github.com/tildeio/rsvp.js/>`_ allows you to bind an event handler to a `global error event
+<https://github.com/tildeio/rsvp.js#error-handling>`_:
+
+.. code-block:: javascript
+
+    RSVP.on('error', function(reason) {
```

Comment:
Oh, also, lol, this is no longer true because `Raven` is a class instance and must be the value of `this` when `captureException` is invoked.

This was another breaking API change.